README.md

```diff
@@ -31,8 +31,10 @@ Building a factorization model for the Movielens 100K dataset is very simple
 ([Colab](https://tensorflow.org/recommenders/examples/quickstart)):
 
 ```python
+import tensorflow as tf
 import tensorflow_datasets as tfds
 import tensorflow_recommenders as tfrs
+from typing import Dict, Text
```

Comment:
Thanks! Could you move the `typing` import to the first line, then leave one empty line, then have `import tensorflow as tf`?